src/backend/base/langflow/components/deactivated/vectara_self_query.py

```
@@ -0,0 +1,77 @@
+
+    @check_cached_vector_store
+    def build_vector_store(self) -> Vectara:
+        """Builds the Vectara Self Query Retriever."""
+        try:
+            from langchain_community.vectorstores import Vectara  # noqa: F401
+        except ImportError as e:
+            msg = "Could not import Vectara. Please install it with `pip install langchain-community`."
+            raise ImportError(msg) from e
+
+        metadata_field_obj = []
+        for meta in self.metadata_field_info:
```

Pylint (3.3.7)

[error] Instance of 'VectaraSelfQueryRetriverComponent' has no 'metadata_field_info' member

(E1101)